commons/data_access_layer/cosmos_db.py

```diff
@@ -133,6 +133,13 @@ def create_sql_condition_for_visibility(
             return 'AND NOT IS_DEFINED(%s.deleted)' % container_name
         return ''
 
+    @staticmethod
+    def create_sql_condition_for_active( only_active: bool,
+        container_name='c' ) -> str:
+        if only_active:
+            return 'AND NOT IS_DEFINED(%s.status) OR (IS_DEFINED(%s.status) AND %s.status = \'active\')' % (container_name, container_name, container_name)
```

Comment:
This line is too long, can you consider put all inside () and add an Enter after '%' character? 👀

Reply:
Try using the .format() method instead for readability.
 ex.
```
tool="Unsupervised algorithms"
goal="patterns"
print("{title} try to find {aim} in the dataset".format(title=tool, aim=goal))
```

Reply:
Ok, It has been adjusted, using a new approach for avoiding the only_active argument.